### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by karin-on**

### CODE REVIEW

File: src/components/Game/Game.scss [hunk 1]
```diff
@@ -3,26 +3,73 @@
...
+}
+
+@keyframes load {
+    0% {
+        transform: rotate(0deg);
+    }
+    100% {
+        transform: rotate(360deg);
+    }
+}
```
Comment (tomecko) [inline src/components/Game/Game.scss:53]: that has to be fun!!!

File: src/components/Game/Letters/Letters.js [hunk 2]
```diff
@@ -9,9 +10,12 @@ const Letters = ({ letters, clicked, dragDisabled, dragEnd }) => (
     <DragDropContext onDragEnd={dragEnd}>
         <section className={classes.Letters}>
             <Droppable droppableId="droppable" direction="horizontal">
-                {(provided) => (
+                {(provided, snapshot) => (
                     <ul
-                        className={classes.LettersContainer}
+                        className={classNames(
+                            classes.LettersContainer,
+                            snapshot.isDraggingOver ? classes.DraggingOver : null,
```
Comment (tomecko) [inline src/components/Game/Letters/Letters.js]: or probably `{ [classes.DraggingOver]: snapshot.isDraggingOver }`
but it's okay right now :)

File: src/components/Game/Slots/Slots.js [hunk 3]
```diff
@@ -12,7 +13,13 @@ const Slots = ({ currentAnswer }) => (
                 .map((i) => currentAnswer[i])
                 .map((letter, index) => (
                     /* eslint-disable react/no-array-index-key */
-                    <li key={index} className={classes.Slot}>
+                    <li
+                        key={index}
+                        className={classNames(
+                            classes.Slot,
+                            !letter ? classes.Empty : null,
```
Comment (tomecko) [inline src/components/Game/Slots/Slots.js]:
```suggestion
                            letter ? null : classes.Empty,
```

File: src/styles/abstracts/_variables.scss [hunk 4]
```diff
@@ -3,8 +3,18 @@ $color-gray-bg: #eee;
...
 $color-black: #000;
+$color-yellow-1: #ffecb7;
+$color-yellow-2: #eedaa3;
+$color-brown-font: #551b05;
+$color-brown-font-disabled: #551c06;
+$color-green-disabled: #8dad84;
+$color-green-main: #06836d;
+$color-green-home-bg: #067c67;
+$color-green-dark-1: #004636;
+$color-green-dark-2: #00311e;
```
Comment (tomecko) [inline src/styles/abstracts/_variables.scss:14]: are those colors in any way related? for example is `$color-green-dark-1` the darker version of `$color-green-main`?
if so I would consider defining main colors only and calculating further colors based on these main colors
sass can manipulate colors: https://sass-lang.com/documentation/modules/color
then it's easier to tune/update/replace color theme

**Review by tomecko — APPROVED** [review]

File: src/components/Game/Game.scss (hunk 1 above)
Comment (karin-on) [reply src/components/Game/Game.scss:53]: copied from the internet :)

File: src/styles/abstracts/_variables.scss (hunk 4 above)
Comment (karin-on) [reply src/styles/abstracts/_variables.scss:14]: I don't know 🤷🏻‍♀️ I got them from a friend

**Review by tomecko — APPROVED** [review]

**Pull request merged**